src/components/molecules/Navigation/Navigation.tsx

```diff
@@ -17,7 +17,7 @@ type NavigationProps = {
 const Navigation = memo(({ sidebar, sticky = false }: NavigationProps) => {
```

Comment:
Look what I found out, when you scroll in the table 😬 
![image](https://user-images.githubusercontent.com/2675250/128019604-efd1804f-3005-4ab7-82d0-32ced430e4d0.png)

Reply:
this is fixed now